### ISSUE TITLE
How to compile Boost.Python

### ISSUE THREAD

**Issue by twalenzyk** [blackberry/Boost#12]
I want to compile Boost.Python but this seems to be difficult (according to the comment in blackberry-config.jam)

Each BB10 device comes with Python3.2.2 installed. 
Does that at least answer the question, if we need a port of python for blackberry?

**Comment by tvaneerd** [comment]
"Each BB10 device comes with Python3.2.2 installed."

I'm not sure if that is always true.  But if it is, then you probably don't need to build python.  But I think you still need python.h and things like that (that come from python, not boost, IIRC).  And it needs to be one for the right platform (ie QNX) not whatever you find on your host platform (Windows/Mac/etc).  But I could be completely wrong - I'm just going from memory here.

But I think a Linux one should be close enough to work.

Basically you are breaking new ground - no one (AFAIK) inside BlackBerry needed Boost Python, so no one ever built it.  Most other pieces of Boost were either used or header-only (thus either built by necessity or easy/nothing to build).

**Comment by twalenzyk** [comment]
Thanks Tony.

I will give it a try.

By the way, I have never used Boost before, nor have I compiled it myself.
It is possible to build only Boost.Python without all the other packages?
